README.md

```diff
@@ -33,4 +33,4 @@ modules: [
 ]
 ```
 ### Update
-Navigate to the folder of the this module in the `/modules` folder and get the latest version using the command `git pull`.
\ No newline at end of file
+Navigate to the folder of the this module in the `/modules` folder and get the latest version using the command `git pull`.
```

Comment:
One more typo here: `the module` or `this module` but not both 🚽

Reply:
I didn't notice that :D